README.md

```diff
@@ -13,27 +13,18 @@ A program for organizing Endless Sky installations.
 ESLauncher2 requires DirectX 11 or 12, Vulkan or Metal. See also [#4](https://github.com/EndlessSkyCommunity/ESLauncher2/issues/4).
 
 
-### Installation instructions for Mac
-#### Additional requirements for Mac ####
-**Important**: currently, the launcher works only when run from a console which has access to the full filesystem.
-We are working on a solution to overcome this.
+### Additional instructions for Mac
```

Comment:
Wether in this PR or not, we should probably add some short Instructions, since these are Additional Instructions.